priv/templates/extended_bin

```diff
@@ -674,8 +690,17 @@ EPMD_MODULE="$(grep '^-epmd_module' "$VMARGS_PATH" || true)"
 if [ "$EPMD_MODULE" ]; then
     DIST_ARGS="${DIST_ARGS} ${EPMD_MODULE}"
 fi
+INET_DIST_USE_INTERFACE="$(grep '^-kernel  *inet_dist_use_interface' "$VMARGS_PATH" || true)"
```

Comment:
Why the `*`?

Reply:
Just to cope with the user adding superflous whitespace in `vm.args`.  I can totally live without that :smile:

Reply:
I mean additional spaces between `-kernel` and `inet_dist_use_interface`.

Reply:
I'd at least use it in quotes if it were possible, to clarify the intent.

Reply:
Sorry I probably need more coffee, you'd add quotes where exactly?

Reply:
Ah nevermind, I misread the regex and nested argument. The quotes are already there. I had initially read it as `'grep ...'` rather than `grep '...'`